UserAdminBundle/Form/Type/UserType.php

```diff
@@ -13,13 +14,21 @@
 class UserType extends AbstractType
 {
     protected $class;
+    protected $languages;
 
     /**
      * @param string $class
      */
-    public function __construct($class)
+    public function __construct($class, Twig_Environment $twig)
```

Comment:
you should use this parameter : `open_orchestra_base.languages.availables` of the symfony kernel and pass it directly to the class, this will be a more cleaner way that to get the `Twig_Environment`